test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -754,7 +754,7 @@ describe('Dropdown Component', () => {
   describe('options', () => {
     it('adds the onClick handler to all items', () => {
       wrapperShallow(<Dropdown options={options} selection />)
-        .children('DropdownItem')
+        .find('DropdownItem')
```

Comment:
This spec was silently broken since this returned an empty list and `everyWhere` on an empty list is passing. I noticed because I used this as a starting point for the spec I added below.